<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellDialog</class>
 <widget class="QDialog" name="SellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sell</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QDateEdit" name="dateTimeEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 87 14pt &quot;Segoe UI Black&quot;;
color: rgb(234, 67, 53);</string>
           </property>
           <property name="displayFormat">
            <string>dd-MMM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 85, 255);
background-image: url(:/images/img/grren_login_screen_by_h2o4life-d3181xz.png);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;786 Plastics&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="font">
              <font>
               <family>Segoe UI Semibold</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(64, 66, 68);</string>
             </property>
             <property name="text">
              <string>Stock</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="totalstock_edit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>25</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(66, 133, 244);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Buyer</string>
                 </property>
                 <property name="buddy">
                  <cstring>dealer_edit</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Material</string>
                 </property>
                 <property name="buddy">
                  <cstring>material_combo</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Cost Per unit</string>
                 </property>
                 <property name="buddy">
                  <cstring>cost_spinbox</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Quantity</string>
                 </property>
                 <property name="buddy">
                  <cstring>quantity_spinbox</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Total</string>
                 </property>
                 <property name="buddy">
                  <cstring>total_spinbox</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Cash Paid</string>
                 </property>
                 <property name="buddy">
                  <cstring>cashpaid_spinbox</cstring>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLineEdit" name="dealer_edit">
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>16</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(1, 55, 65);
color: rgb(230, 230, 230);</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="material_combo">
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="cost_spinbox">
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>99999999</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="quantity_spinbox">
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>99999999</number>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QSpinBox" name="total_spinbox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>20</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(64, 66, 68);
color: rgb(146, 189, 108);</string>
                   </property>
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>99999999</number>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="toolButton">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSpinBox" name="cashpaid_spinbox">
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(186, 96, 0);</string>
                 </property>
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>99999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListView" name="dealers_view"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="deal_button">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Deal</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reset_button">
               <property name="text">
                <string>Reset</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>325</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>38</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Balance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="debt_label">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(234, 67, 53);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="status_label">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>(+) status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>material_combo</tabstop>
  <tabstop>cost_spinbox</tabstop>
  <tabstop>quantity_spinbox</tabstop>
  <tabstop>cashpaid_spinbox</tabstop>
  <tabstop>reset_button</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>totalstock_edit</tabstop>
  <tabstop>total_spinbox</tabstop>
  <tabstop>dealers_view</tabstop>
  <tabstop>deal_button</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>dealer_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
